boards/nxp/ls1046ardb/doc/index.rst

```diff
@@ -91,7 +91,7 @@ kernel tests on LS1046A RDB board. For example, with the :zephyr:code-sample:`sy
 .. zephyr-app-commands::
    :zephyr-app: samples/synchronization
    :host-os: unix
-   :board: nxp_ls1046ardb
+   :board: nxp_ls1046ardb/ls1046a
```

Comment:
the SoC can be excluded from these build commands since there is only one

Reply:
Fixed, thanks.